asciiracer/multiplayer/client.py

```diff
@@ -0,0 +1,84 @@
+import logging
+import json
+import threading
+
+from websocket import WebSocketApp
+
+from asciiracer.config import SERVER_HOST, SERVER_PORT
+from asciiracer.multiplayer.enums import Events
+
+
+class Client(WebSocketApp):
+    def __init__(self, logger=logging.getLogger(__name__)):
+        self.logger = logger
+        self.score_table = {}
+        self.client_running = False
+        self.id = None
+        self.last_score = 0
+        super().__init__(
+            url=f"ws://{SERVER_HOST}:{SERVER_PORT}",
+            on_open=self.on_open,
+            on_message=self.on_message,
+            on_error=self.on_error,
+            on_close=self.on_close
+        )
+
+    def run_client_in_thread(self):
+        threading.Thread(
+            target=self.run_forever,
+            daemon=True
+        ).start()
+        while not self.client_running:
+            ...
+        return
+
+    def handle_event(self, event: dict):
+        if event["event_type"] == Events.update_score.value:
+            self.score_table = event["value"]
+
+        elif event["event_type"] == Events.id_message.value:
```

Comment:
Enums can be checked for equality by `is`:
```suggestion
        event_variant = Events(event["event_type"])
        if event_variant is Events.update_score:
            self.score_table = event["value"]

        elif event_variant is Events.id_message:
```